build-tests/api-extractor-test-04/etc/api-extractor-test-04.api.md

```diff
@@ -83,6 +83,24 @@ export interface IPublicComplexInterface {
 
 export { Lib1Interface }
 
+declare namespace NS {
+  export {
+    NS_PUBLIC,
+    NS_BETA,
+    NS_INTERNAL
+  }
+}
+export { NS }
+
+// @beta (undocumented)
+const NS_BETA = "BETA";
+
+// @internal (undocumented)
+const NS_INTERNAL = "INTERNAL";
```

Comment:
The test is not actually generating an internal rollup and therefore it does not verify trimming of `NS_INTERNAL`

Reply:
Fixed in 357d626a55a62540b897e866ec428f8dcfc6b6a7